config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
-  resources :issues
+  resources :issues	
+  get '/issues/:id/prepare', to: 'issues#prepare', as: 'issue_prepare_form'
+	patch '/issues/:id/prepare', to: 'issues#prepare', as: 'issue_prepare_update'
```

Comment:
Dobry kierunek. Trzeb jednak zrobić to wewnątrz bloku `do..end` a poza tym wolałbym wyodrębnić dwie osobne metody w kontrolerze `issues`.